recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
   run:
     - python
     - cloudpickle >=1.6.0,<=2.0.0
-    - snowflake-connector-python
+    - snowflake-connector-python>=2.7.4
     - typing-extensions >=4.1.0
   run_constrained:
     - pandas >1,<1.4
```

Comment:
Since `pip` is a dependency for the package, we probably should include `pip` in the `test section`  
https://github.com/AnacondaRecipes/snowpark-python-feedstock/blob/v080-bump-2/recipe/meta.yaml#L31&L34

Reply:
@varlackc I'm not sure I understand. `pip` is not a dependency for the package as far as I'm aware.
Furthermore note that we don't just have the `setup.py` to go by, but also Snowflake's own conda recipe:
https://github.com/snowflakedb/snowpark-python/blob/main/recipe/meta.yaml

Reply:
OK, based on other packages' patterns I have added pip as a test dependency and `pip check` as a test command

Reply:
@varlackc do you think this is what you had in mind?

Reply:
`pip` is needed here only in `host` and also in `test/requires` for using `pip check`